ui/src/app/utils/access-utils.tsx

```diff
@@ -459,6 +547,20 @@ export const getAccessModuleStatusByName = (
   );
 };
 
+// Gets status of moduleName, but if it is an identity module,
+// it will return the status of the "most complete" identity module.
+export const getRelativeAccessModuleStatus = (
```

Comment:
If someone can think of a better name for this function, let me know.

This function exists, because sometimes we need to look up the exact status of a module, but in other cases (like the data access requirements page), we need to. know the status of a group of modules (login.gov and id.me). This function handles the second case.